src/state/learnocaml_data.mli

```diff
@@ -171,6 +171,8 @@ module Exercise: sig
 
     val enc: t Json_encoding.encoding
 
+    val sub_enc: t Json_encoding.encoding
```

Comment:
As discussed in our meeting, you should be able to remove that line (enforcing that there is a `learnocaml_version: "2"` in subindex.json? - I've just updated https://github.com/ocaml-sf/learn-ocaml/issues/331#issuecomment-842543557)